2 - RTB/Documentazione esterna/Verbali/23-12-15/23-12-15.typ

```diff
@@ -0,0 +1,59 @@
+#import "/template.typ": *
+
+#show: project.with(
+  date: "15/12/23",
+  subTitle: "",
+  docType: "verbale",
+  authors: (
+    "Nardo Silvio",
+  ),
+  externalParticipants : (
+    (name: "Alberton Michela", role: "Membro azienda proponente"),
+    (name: "Bassani Matteo", role: "Referente aziendale"),
+    (name: "Ruffin Filippo", role: "Membro azienda proponente"),
+  ), 
+  timeStart: "12:00",
+  timeEnd: "12:30",
+);
+
+= Ordine del giorno
+Ai referenti aziendali sono stati esposti i progressi sui PoC rispetto al precedente meeting esterno e sono sorte discussioni relativamente a:
+- Analisi dei Requisiti;
+- tecnologie da utilizzare e utilizzate;
+- feature dei PoC;
+- meeting futuri.
+
+== Considerazioni sull'Analisi dei Requisiti
+Non sono stati identificati particolari vincoli sulle performance.
+
+=== Vincoli sui browser 
+È richiesta la compatibilità dell'applicazione almeno con i browser:
+- Chrome (da versione 89);
+- Firefox;
```

Comment:
Mancano i numeri di versione dei browser

Reply:
io ho riportato i vincoli dei browser in base a quanto approvato dal dialogo con il proponente, alla domanda lui non ha specificato una versione a parte per chrome

Reply:
Come nel commento precedente, i proponente può al massimo confermare quanto da nooi individuato in merito alle versioni. Si è parlato di Chrome, è vero, ma nella nostra analisi sono già presenti anche le versioni minime per gli altri browser.